plugins/bulk-import-backend/catalog-info.yaml

```
@@ -23,3 +23,31 @@ spec:
   owner: rhdh-team
   system: rhdh
   subcomponentOf: janus-idp-bulk-import
+---
```

```suggestion
  providesApis:
    - janus-idp-bulk-import-backend-api

---
```

Should we add a `providesApi` relation between the Component and API entities, just like https://github.com/janus-idp/backstage-plugins/blob/main/plugins/ocm-backend/catalog-info.yaml#L29C3-L30 ?

---

Oh pardon, forgot this. Fixed.